### ISSUE TITLE
SPI mode must be 1

### ISSUE THREAD

**Issue by Ny4nmaru** [dhhagan/py-opc#59]
Hi, i have a problem with using this library. i import opc and usbiss successfully, i even create a spi object as written in documentation, but i have problem with "alpha".

when i write opc.OPCN2(spi), it returns error "assert self.cnxn.mode ==1 "SPI mode must be 1"

i am using python 3.5.3, running OPC-N2 via SPI-USB adapter that was supplied with the device. Trying to run it on Raspberry Pi 3 model B.

Do you have any advice ?:)

**Comment by dhhagan** [comment]
Hey @Ny4nmaru, thanks for reaching out. Could you post the exact code you are using to setup the SPI connection? Could you also post the versions of py-opc and pyusbiss that you have installed? You can grab those by issuing the command `pip freeze` at the terminal.

**Comment by DancingQuanta** [comment]
Hi,, I am the author of pyusbiss library. The library has been suffering from my lack of care with releases. As such the documentation I gave to David about pyusbiss is out of date.

Please give me a few days while I get the library under proper versioning and release a stable version then I will update the documentation. In the meantime try the following code
```
spi = usbiss.USBISS("/dev/ttyACM0", 'spi', spi_mode = 1, freq = 500000)
```
Please note that the correct code includes the statement
```
spi_mode = 1
```
while the old code is
```
spi_mode = 2
```
This change was made to bring pyusbiss in line with the official SPI specification.

**Comment by dhhagan** [comment]
Huh. That is strange. Have you had any more issues, or has it been cooperating?

**Comment by Ny4nmaru** [comment]
Well, i downloaded both needed libraries again, and i found out, that the "SPI mode must be 1" row is deleted in the fully functional library. nothing else is different. And what is really weird is that i dont remember deleting it. But whatever :D its funny, but without that row its working like a charm. OPC-N2 was with few pauses measuring for 12h, without any problem.

**Comment by Ny4nmaru** [comment]
I guess that sometimes science is more art than science

**Comment by DancingQuanta** [comment]
I have been working hard behind the scenes. :). If everything works for you then you can close this issue.
Also I am releasing a minor update soon that changes the interface which will allow more development into other capacities of the USB-SPI adapter.

**Comment by DancingQuanta** [comment]
Hello, I have released a new version of `pyusbiss` 0.2.0 which you can install via `pip install pyusbiss`. This version is a major revision and so I have updated the instructions in https://github.com/dhhagan/py-opc/pull/61.
